README.md

```diff
@@ -8,23 +8,47 @@ Service manager port: 10123
 ### Testing
 
 Run unit tests:
-<pre>sbt test</pre>  
+<pre>sbt test</pre>
 Run integration tests:
 <pre>sbt it/test</pre>
 Run accessibility linter tests:
 <pre>sbt A11y/test</pre>
 
 ### Running manually or for journey tests
 
+To toggle between the Phase 5 transition and post-transition modes we have defined two separate modules, each with their own set of class bindings to handle the rules associated with these two periods.
+
+#### Transition
+<pre>
+sm2 --start CTC_TRADERS_P5_ACCEPTANCE_TRANSITION
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_UNLOADING_FRONTEND
```

Comment:
Should be `MANAGE_TRANSIT_MOVEMENTS_UNLOADING_FRONTEND_TRANSITION`

Reply:
Fixed